docs/02-app/02-api-reference/04-functions/unstable_cache.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: unstable_cache
+description: API Reference for the unstable_cache function.
+---
+
+`unstable_cache` allows you to cache the results of expensive operations, like database queries, and reuse them across multiple requests.
+
+```jsx
+import { getUser } from './data';
+import { unstable_cache } from 'next/cache';
+
+const getCachedUser = unstable_cache(
+  async (id) => getUser(id),
+  ['my-app-user']
+);
+
+export default async function Component({ userID }) {
+  const user = await getCachedUser(userID);
+  ...
+}
+```
+
+> **Warning**: This API is unstable and may change in the future. We will provide migration documentation and codemods, if needed, as this API stabilizes.
+
+## Parameters
+
+```jsx
+const data = unstable_cache(fetchData, keyParts, options)()
+```
+
+- `fetchData`: This is an asynchronous function that fetches the data you want to cache. It must be a function that returns a `Promise`.
+- `keyParts`: This is an array that identifies the cached key. It must contain globally unique values that together identify the key of the data being cached. The cache key also includes the arguments passed to the function.
+- `options`: This is an object that controls how the cache behaves. It can contain the following properties:
+  - `tags`: An array of tags that can be used to control cache invalidation.
+  - `revalidate`: The number of seconds after which the cache should be revalidated.
```

Comment:
For clarities sake, it would be nice to provide another code example (in addition to the one at the top of the page) after this list of params, that uses all params, with code comments to explain each part.

Reply:
Good idea!